### PR TITLE
Release: 3.2.0

### PR DESCRIPTION
**Pull request by EdwardLarson**
- [X] I have reviewed the [OFRAK contributor guide](https://ofrak.com/docs/contributor-guide/getting-started.html) and attest that this pull request is in accordance with it.

### Added
- Add a JFFS2 packer and unpacker. ([#326](https://github.com/redballoonsecurity/ofrak/pull/326))
- Add method to Resource and data service to search for patterns in its data ([#333](https://github.com/redballoonsecurity/ofrak/pull/333))
- Add search bars to GUI in order to search for a string or bytes within a resource. ([#345](https://github.com/redballoonsecurity/ofrak/pull/345))
- Add Identifier, Unpacker, Packer for Intel Hex format. ([#349](https://github.com/redballoonsecurity/ofrak/pull/349))
- Add unpackers for EXT filesystems (versions 2 through 4). ([#337](https://github.com/redballoonsecurity/ofrak/pull/337))
- A new feature that allows users to create an OFRAK "project" that contains a collection of scripts and binaries. ([#360](https://github.com/redballoonsecurity/ofrak/pull/360))

### Changed
- Support uploading files in chunks to handle files larger than 2GB from the GUI ([#324](https://github.com/redballoonsecurity/ofrak/pull/324))
- Do not look for the `ofrak_patch_maker` toolchain.conf in /etc so local installs do not need escalated permissions. ([#342](https://github.com/redballoonsecurity/ofrak/pull/342))

### Fixed
- Save resources affected by data patches and dependency updates on a resource being saved ([#355](https://github.com/redballoonsecurity/ofrak/pull/355))

### CODE REVIEW

File: ofrak_core/CHANGELOG.md [hunk 1]
```diff
@@ -4,17 +4,22 @@ All notable changes to `ofrak` will be documented in this file.
...
 - Add a JFFS2 packer and unpacker. ([#326](https://github.com/redballoonsecurity/ofrak/pull/326))
 - Add method to Resource and data service to search for patterns in its data ([#333](https://github.com/redballoonsecurity/ofrak/pull/333))
 - Add search bars to GUI in order to search for a string or bytes within a resource. ([#345](https://github.com/redballoonsecurity/ofrak/pull/345))
 - Add Identifier, Unpacker, Packer for Intel Hex format. ([#349](https://github.com/redballoonsecurity/ofrak/pull/349))
+- Add unpackers for EXT filesystems (versions 2 through 4). ([#337](https://github.com/redballoonsecurity/ofrak/pull/337))
 - A new feature that allows users to create an OFRAK "project" that contains a collection of scripts and binaries. ([#360](https://github.com/redballoonsecurity/ofrak/pull/360))
 
 ### Changed
 - Support uploading files in chunks to handle files larger than 2GB from the GUI ([#324](https://github.com/redballoonsecurity/ofrak/pull/324))
+- Do not look for the `ofrak_patch_maker` toolchain.conf in /etc so local installs do not need escalated permissions. ([#342](https://github.com/redballoonsecurity/ofrak/pull/342))
```
Comment (whyitfor) [inline ofrak_core/CHANGELOG.md:19]: Isn't this a chance in ofrak_patch_maker and not ofrak?

**Comment by whyitfor** [comment]
Right now the dependencies are:
```
install_requires=[
        "ofrak_io>=1.0,==1.*",
        "ofrak_type>=2.2.0rc0,==2.*",
        "ofrak_patch_maker>=4.0.1rc0,==4.*",
    ]
```

Is there a reason these can't be >= to the currently released versions of these packages?

**Review by whyitfor — COMMENTED** [review]
See comment. Were you planning on releasing ofrak patch maker 4.1.0 with this release?

File: ofrak_core/CHANGELOG.md (hunk 1 above)
Comment (EdwardLarson) [reply ofrak_core/CHANGELOG.md:19]: Good point. I think this will bump ofrak_patch_maker to 4.0.2

**Pull request merged**